metrics/metrics.go

```diff
@@ -104,6 +105,29 @@ var (
 		// TODO(soltesz): support a status field based on HTTP status.
 		[]string{"worker"},
 	)
+
+	FileSizeHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name: "etl_web100_snaplog_file_size_bytes",
+			Help: "Size of individual snaplog files.",
```

Comment:
Note: We should probably generalize this to work for any data type, but just snaplog is fine for now.